client/components/domains/free-domain-explainer/index.jsx

```diff
@@ -45,19 +45,25 @@ class FreeDomainExplainer extends React.Component {
 	}
 
 	render() {
-		const { translate, isReskinned } = this.props;
+		const { translate, isReskinned, locale } = this.props;
 		const { TextWrapper } = this;
 
 		return (
 			<div className="free-domain-explainer card is-compact">
 				<header>
 					<h1 className="free-domain-explainer__title">
-						{ translate( 'Get a free one-year domain registration with any paid plan.' ) }
+						{ locale === 'en'
+							? translate( 'Get a free one-year domain registration with any paid annual plan.' )
```

Comment:
> " ... with any paid annual plan."

Just to check, is this change supposed to be only for users in the monthly test treatment group? This currently applies to all users. Not a big deal I guess even if we changed that one word for control group users too.